packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -24,11 +25,16 @@ type PersistOptions = Pick<
 	| 'r2Persist'
 >;
 
-export function getPersistence(persistPath: string | false): PersistOptions {
-	if (persistPath === false) {
+export function getPersistence(
+	root: string,
+	persistState = true,
```

Comment:
Do we ever call this with `persistState` set to false?

Reply:
Not at the moment. I would be inclined to add `persistState` as an option to the plugin config so left the option open. I think people didn't feel this was necessary when we discussed it the other day. Happy to go with what you prefer here.

No need for this function to be exported so I'll remove that.